### ISSUE TITLE
Pretty formatting shouldn't be enabled in Development mode by default

### ISSUE THREAD

**Issue by matthewbjones** [slim-template/slim-rails#78]
The change at this commit a845a7b6fdfb3d928ee3e6c7f72fc5c24ce3737e introduced a problem where there is a difference between development mode and production mode. Normally, pretty-printing in development mode should have no impact on behavior from production mode, however, this is not the case.

With pretty mode enabled on slim-rails 3.0.1, the resulting HTML code is incorrectly generated, leading to rendering problems. Specifically, if you have the following slim code:

```slim
span before
span = " middle "
span after
```

it results in the following incorrect HTML:

```html
<span>before</span><span>middle </span><span>after</span>
```

Note the missing leading space from the middle span. This causes the webpage to render the text as:

```
beforemiddle after
```

Whereas, if the pretty mode was disabled (like it is for production), the following HTML is generated:

```html
<span>before</span><span> middle </span><span>after</span>
```

resulting in the rendering of

```
before middle after
```

Although the pretty formatting issue comes from a dependency gem, I believe that opting everyone into pretty-formatting mode by default may be problematic. I will try to locate the pretty formatting bug within the 'slim' gem or one of its dependencies but I wanted to file this issue here as well. Any guidance on where to fix the problem would be appreciated.

**Comment by minad** [comment]
The indentation problem was fixed by temple 0.7.5

**Comment by matthewbjones** [comment]
Awesome, thanks for the fix
